crates/ln-dlc-node/src/node/dlc_manager.rs

```diff
@@ -64,3 +68,26 @@ pub fn signed_channel_state_name(signed_channel: &SignedChannel) -> String {
 
     name.to_string()
 }
+
+impl<P> Node<P>
+where
+    P: Send + Sync,
+{
+    pub fn get_signed_channel_by_trader_id(&self, trader_id: PublicKey) -> Result<SignedChannel> {
+        let channel = self.get_dlc_channel_signed(&trader_id)?.with_context(|| {
+            format!("Expect signed dlc channel to exist. trader_id={trader_id}")
```

Comment:
🙃 The way you wrote this originally sounded like an `expect`. 

```suggestion
            format!("Could not find signed DLC channel. trader_id={trader_id}")
```